Makefile

```
@@ -1,11 +1,23 @@
 SOURCE_FILES := $(shell find . -type f -name '*.go')
+VERSION := $(shell git describe --tags --abbrev=0 | cut -c2-)
```

What about doing something like that:

```suggestion
VERSION ?= $(shell git describe)
```

Reasons:
* `VERSION` can be overridden via an environment variable. I can imagine doing that right before tagging a new release
* The vanilla `git describe` provides information about the closest tag + the HEAD of the repo. This is nice for annotations during the development phase

---

I agree that the `VERSION` can be override. But I'm not sure if we need to update the command to the standard output of the `git describe`. This value is used by `kwctl` to build the image tag and the download URL. Therefore, we must tag the branch and release it over the CI jobs to make that [work](https://github.com/kubewarden/github-actions/blob/d849020c9137340c2373d1cbc9cc571b2b18c17e/policy-release/action.yaml#L48) end to end. But we already cover that using the `latest` tag during development phase. It has the latest changes in the [branch](https://github.com/kubewarden/github-actions/blob/d849020c9137340c2373d1cbc9cc571b2b18c17e/policy-release/action.yaml#L28-L39). In other words, if we change the command, we can end in a scenario where the artifact hub points to a image that does not exist. To solve that, it will require changes in our CI job and I do not believe that we need to do that now.

---

@flavio and I have talked in other communication channels and decided that my previous comment is not a problem to the suggested change. Therefore, I've pushed a code with it.